pkg/engine/testdata/lambda_efs.expect.yaml

```diff
@@ -164,7 +164,7 @@ resources:
     aws:efs_mount_target:test-efs-fs:subnet-1-test-efs-fs:
         FileSystem: aws:efs_file_system:test-efs-fs
         SecurityGroups:
-            - aws:security_group:vpc-0:subnet-1-test-efs-fs
+            - aws:security_group:vpc-0:lambda_test_app-test-efs-fs
```

Comment:
it changed back to what was expected. The EFS Mount targets should be sharing a security group since we know that they are receiving traffic from the same subnets